content/200-concepts/300-more/300-telemetry.mdx

```diff
@@ -28,8 +28,8 @@ Data is collected in two scenarios that are described below.
 
 Invocations of the `prisma` CLI and general usage of Studio results in data being sent to the telemetry server at https://checkpoint.prisma.io. Note that:
 
-- Information is only sent every 12 hours (sending is paused for 12 hours after any invocation)
 - The data does **not** include your schema or the data in your database
+- Information is only sent from Prisma CLI after executing a CLI command
```

Comment:
```suggestion
- Prisma only sends information after it executes a CLI command
```

Reply:
Prisma does not execute a CLI command, the user does:
```suggestion
- Information is only sent from Prisma CLI after you executed a CLI command
```
Does this work for you?

Reply:
OK, I'll modify my suggestion. I would like to keep "Prisma sends" though, to avoid passive voice.